develop/blockchains/get-started/intro-polkadot-sdk.md

```diff
@@ -0,0 +1,86 @@
+
+## Polkadot SDK Overview
+
+The Polkadot SDK is composed of five major components:
+
+![](/images/develop/blockchains/get-started/intro-polkadot-sdk/polkadot-sdk-structure.webp)
+
+- [Substrate](https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/polkadot_sdk/substrate/index.html){target=\_blank} - a set of libraries for building blockchains
+- [FRAME](https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/polkadot_sdk/frame_runtime/index.html){target=\_blank} - a blockchain development framework built on top of Substrate
+- [Cumulus](https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/polkadot_sdk/cumulus/index.html){target=\_blank} - a set of libraries and pallets to add parachain capabilities to a Substrate/FRAME runtime
+- [XCM (Cross Consensus Messaging)](https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/polkadot_sdk/xcm/index.html){target=\_blank} - the primary format for conveying messages between parachains 
+- [Polkadot](https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/polkadot_sdk/polkadot/index.html){target=\_blank} - the node implementation for the Polkadot protocol
+
+### Substrate
+
+Substrate is a Software Development Kit (SDK) that uses Rust-based libraries and tools to enable you to build application-specific blockchains from modular and extensible components. Application-specific blockchains built with Substrate can run as standalone services or in parallel with other chains to take advantage of the shared security provided by the Polkadot ecosystem. Substrate includes default implementations of the core components of the blockchain infrastructure to allow you to focus on the application logic.
+
+Every blockchain platform relies on a decentralized network of computers—called nodes—that communicate with each other about transactions and blocks.
+In general, a node in this context is the software running on the connected devices rather than the physical or virtual machine in the network.
+As software, Substrate-based nodes consist of two main parts with separate responsibilities:
+
+- **Client** - services to handle network and blockchain infrastructure activity
+    - Native Binary
+    - Executes the Wasm runtime
+    - Everything else: Database, Networking, Mempool, Consensus, and more
+    - Also known as Host
+- **Runtime** - business logic for state transitions
+    - Application logic
+    - Wasm
+    - Stored as a part of the chain state
+    - Also known as STF (State Transition Function)
+
+### FRAME
+
+FRAME provides the core modular and extensible components that make the Substrate SDK flexible and adaptable to different use cases.
+FRAME includes Rust-based libraries that simplify the development of application-specific logic.
+Most of the functionality that FRAME provides takes the form of plug-in modules called [pallets](/polkadot-protocol/glossary#pallet) that you can add and configure to suit your requirements.
+
```

Comment:
Here we could also use a diagram like this:
<img width="936" alt="image" src="https://github.com/user-attachments/assets/4bdbf23f-7b01-41af-9146-c72e2ee18855">
Or
<img width="605" alt="image" src="https://github.com/user-attachments/assets/0a0d4def-3801-4bfd-8a7c-efcf71a5d4fc">

Reply:
This one is nicer. Do you think we need this before launch or can it go on the list for after?

Reply:
We can add it later. I've created tickets in Jira